include/deal.II/fe/fe.h

```diff
@@ -149,6 +149,65 @@ template <int dim, int spacedim> class FESystem;
  * FiniteElement::system_to_block_index(). The number of blocks of a finite
  * element can be determined by FiniteElement::n_blocks().
  *
+ * To better illustrate these concepts, let's consider the following
+ * example
+ * @code
+ * FESystem<dim> fe_basis(FE_Q<dim>(2), dim, FE_Q<dim>(1),1);
+ * @endcode
+ * with <code>dim=2</code>. The resulting finite element has 3 components:
+ * two coming from the quadratic element and one from the linear element. In case
+ * of fluids, one can think of the first two components representing a
+ * vector-valued velocity field whereas the last one being a scalar valued
+ * pressure field. Without DoF renumbering this finite element will
+ * produce the following distribution of local degrees of freedom:
```

Comment:
"degrees-of-freedom", or perhaps just "DoFs" since the abbreviation is now defined above.